src/cmds/hardware/ide.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char **argv) {
 	int opt;
 
 	getopt_init();
```

Comment:
Can you also remove getopt_init(); it's not required now.

Reply:
ok. however there are many places where getopt_init() appears. wouldnt it be better to remove them from all the modules at once? or at least one commit per section (first cmd, then platform, fs, compat, etc)?

Reply:
> ok. however there are many places where getopt_init() appears. wouldnt it be better to remove them from all the modules at once? or at least one commit per section (first cmd, then platform, fs, compat, etc)?

Yes, you are right there are a lot of places, may be it's better to remove it in single commit

Reply:
ok, Ill do that in a separate commit then. if I can ask, which is the new method with which getopt_init() is no longer needed? I see those variables being declared as global, so Im not sure how they are reset before each command execution?

Reply:
I tried just removing all appearances of getopt_init(), and apparently the tests for getopt() arent passing. it's also breaking some other things.

Reply:
I just created an issue for this. #1833